driverapi/driverapi.go

```diff
@@ -106,3 +103,76 @@ type DriverCallback interface {
 type Capability struct {
 	DataScope datastore.DataScope
 }
+
+// IPData represents the per-network ip related
+// operational information libnetwork will send
+// to the network driver during  CreateNetwork()
+type IPData struct {
```

Comment:
Based on a conversation with @mrjana here:
https://botbot.me/freenode/docker-network/2015-09-23/?msg=50364679&page=1
Can you please add addresspace as part of this structure so that CreateNetwork can get a subnet and addresspace which is a unique combination and identifies a subnetid.